Form-Controls/index.html

```diff
@@ -13,15 +13,67 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+          <p>
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="user_name" required minlength="2" maxlength="50" />
+
+          </p>
+          <p>
+            <label for="mail">Email:</label>
+            <input type="email" id="mail" name="user_email" required/>
+          </p>
+          <p>
+            <label for="colour"> What colour should this T-shirt be? </label> <br>
+              <input type="radio" id="green" name="Tshirt_colour" value="Green">
```

Comment:
Please, avoid using capital letters in your attributes. Only lowercase letters and hyphens is the preferred way to code.